app/mailers/app_mailer.rb

```diff
@@ -1,7 +1,12 @@
-class AppMailer < ActionMailer::Base 
+class AppMailer < ActionMailer::Base
   def send_welcome_email(user)
     @user = user 
-    mail to: @user.email_address, from: 'info@myflix.com', subject: 'Welcome to Myflix!'
+    mail to: user.email_address, from: 'info@myflix.com', subject: 'Welcome to Myflix!'
+  end
+  
+  def send_password_reset(user)
+    @user = user
+    mail to: user.email_address, from: 'info@myflix.com', subject: 'Password Reset'
```

Comment:
You can refactor `from: ...` to `default from: ...` - works similar as `before_action` in Controllers.